clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -184,9 +184,8 @@
                                         Importance.HIGH,
                                         HEARTBEAT_INTERVAL_MS_DOC)
                                 .define(PARTITION_ASSIGNMENT_STRATEGY_CONFIG,
-                                        Type.STRING,
-                                        "range",
-                                        in("range", "roundrobin"),
+                                        Type.CLASS,
+                                        RoundRobinAssignor.class,
```

Comment:
Yeah, that's a good point. Maybe we leave it for now and address this for a separate JIRA? I can default to RangeAssignor for consistency with the current version.